CHANGELOG.md

```diff
@@ -8,6 +8,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 TODO: Insert version codename, and username of the contributor that named the release.
 -->
 
+## [0.11.1] - 2022-05-13: Simon's Carefully Chosen Release Name II
+
+Single change which fixed a bug introduced in 0.11.0 which could cause
+unwanted unilateral closes (`bad reestablish revocation_number: 0 vs 3`)
+
+### Fixed
+
+ - connectd: make sure we don't keep stale reconnections around. ([#5256])
+ - connectd: fix assert which we could trigger. ([#5256])
```

Comment:
```suggestion
 - connectd: fix assert which we could trigger. ([#5254])
```

This should be https://github.com/ElementsProject/lightning/issues/5254 ?

Reply:
No, these refer to the PR, not the bug itself...

Reply:
Ah damnt! Sorry just assuming wrong stuff